base/launcher/src/main.cc

```diff
@@ -39,6 +40,14 @@ using namespace motis;
 int main(int argc, char const** argv) {
   motis_instance instance;
 
+  auto reg = subc_reg{};
+  for (auto const& m : instance.modules()) {
+    m->reg_subc(reg);
+  }
+  if (argc >= 1 && !utl::cstr{argv[1]}.starts_with("-")) {
```

Comment:
```suggestion
  if (argc > 1 && !utl::cstr{argv[1]}.starts_with("-")) {
```

Da hier `argv[1]` verglichen wird, ist es denke ich besser, `argc > 1` zu checken, auch wenn es so wohl auch funktioniert.